pcss-criminal-application/src/main/java/ca/bc/gov/open/pcsscriminalapplication/service/HearingValidator.java

```diff
@@ -0,0 +1,9 @@
+package ca.bc.gov.open.pcsscriminalapplication.service;
+
+import ca.bc.gov.open.wsdl.pcss.one.SetHearingRestrictionCriminalRequest;
+
+import java.util.List;
+
+public interface HearingValidator {
+    public List<String> validateSetHearingRestrictionCriminal(SetHearingRestrictionCriminalRequest request);
```

Comment:
This doesn't require the public modifier.

Reply:
I tried removing the public modifier and I'm getting compilation errors. I think it needs to be there.

Reply:
Strange I haven't ever done that.